packages/preferences/src/browser/views/preference-editor-widget.ts

```diff
@@ -333,7 +333,7 @@ export class PreferencesEditorWidget extends BaseWidget implements StatefulWidge
     protected scrollWithoutModelUpdate(node?: HTMLElement): void {
         if (!node) { return; }
         this.shouldUpdateModelSelection = false;
-        node.scrollIntoView();
+        this.scrollContainer.scrollTo(0, node.offsetTop);
```

Comment:
This fixes the problem when I expand the node when the preference editor is open. However, I can still reproduce the problem when I reload the page with the pref editor open and node expanded. I can fix that case if I change the other instance of `scrollIntoView()` in this file (line 245) to 
```
this.scrollContainer.scrollTo(0, renderer.node.offsetTop);
```

Reply:
Good catch 👍